doc/source/reference/functions/between.rst

```diff
@@ -15,10 +15,12 @@ It is often used in combination with :ref:`select-filter` option in :doc:`/refer
 Syntax
 ------
 
-``between`` has three or five parameters::
+``between`` has some arguments from three to six parameters::
```

Comment:
Why do you mix "arguments" and "parameters"? We should not mix "arguments" and "parameters" without particular reason.